inst/template/dust_methods.cpp

```diff
@@ -0,0 +1,77 @@
+/// IMPORTANT; changes here must be reflected in inst/template/dust_methods.hpp
```

Comment:
Can you update these files in the developer notes, I would probably forget to look here

Reply:
done